R/forecast.R

```diff
@@ -459,27 +459,34 @@ is_forecast.default <- function(x, ...) {
 #' @export
 #' @rdname is_forecast
 #' @keywords check-forecasts
-is_forecast.forecast_sample <- function(x, ...) {
-  inherits(x, "forecast_sample")
+is_forecast.forecast <- function(x, ...) {
+  inherits(x, "forecast")
+}
+
+#' @export
+#' @rdname is_forecast
+#' @keywords check-forecasts
+is_forecast_forecast_sample <- function(x, ...) {
```

Comment:
I think this is slightly missing the point? What I was expecting is is_forecast_sample (and so on along with as_forecast_sample maybe coming in a later PR).

Reply:
🤦 
Yeah I was just being tired and silly here... updated!